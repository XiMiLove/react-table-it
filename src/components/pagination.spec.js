import React from 'react';
import Pagination from './pagination';

describe('Pagination Component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Pagination />);
    expect(wrapper).toMatchSnapshot();
  });
});
