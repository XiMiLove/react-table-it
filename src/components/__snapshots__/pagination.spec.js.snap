// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination Component renders correctly 1`] = `ShallowWrapper {}`;
